plugins/conference/lib/model/ConferenceEntryServerNode.php

```diff
@@ -87,4 +87,9 @@ public function setLastAllocationTime($v)
 		$this->putInCustomData(self::CUSTOM_DATA_LAST_ALLOCATE_TIME, $v);
 	}
 
+	public function isValid()
+	{
+		return (time() - kConf::get('conf_not_finished_timeout', 'local', 0)) <= $this->getLastAllocationTime();
```

Comment:
Make sure to add conf_not_finished_timeout default in conf files

Reply:
I suggest adding default value 0 to getLastAllocationTime , otherwise you will be comparing int with null.